Exercise_D.js

```diff
@@ -14,7 +14,16 @@
 Use a combination of conditionals and loops to create your answer for this question. */
 
 function multiplyNumber(number) {
-  // CODE HERE
+  if (number <= 0 || isNaN(number)) {
+    return "ERROR";
+  } if (number >= 1000000) {
+    return number;
+  } else {
+    while (number <= 1000000) {
+      number = number * 10;
+    }
+    return number;
+  };
 }
 

```

Comment:
Fantastic job, only logic that could be 'shortened'
is on line 23

      number = number * 10;
change to
number *= 10